src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingOptions.java

```diff
@@ -39,12 +42,18 @@ public HearingOptions(@JsonProperty("wantsToAttend") String wantsToAttend,
         this.arrangements = arrangements;
         this.scheduleHearing = scheduleHearing;
         this.excludeDates = excludeDates;
+        this.agreeLessNotice = agreeLessNotice;
         this.other = other;
     }
 
     @JsonIgnore
     public Boolean isWantsToAttendHearing() {
-        return wantsToAttend != null && wantsToAttend.toLowerCase().equals("yes");
+        return StringUtils.isNotBlank(wantsToAttend) && wantsToAttend.toLowerCase().equals("yes");
+    }
+
+    @JsonIgnore
+    public Boolean isAgreeLessNotice() {
+        return StringUtils.isNotBlank(agreeLessNotice) && agreeLessNotice.toLowerCase().equals("yes");
```

Comment:
@jackmaloney should this be notEqual  to "no" instead of equal to "yes"?

Reply:
Looks fine